.github/workflows/ReactNativeSlider-CI.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Run ESLint on the sources
         run: cd package && npx eslint src
 
+      - name: Run TypeScript on the sources
+        run: cd package && npx tsc --noEmit
```

Comment:
Don't we want to run tsc on the example app as well?

Reply:
I think we missed that during implementing #427